overlay.py

```diff
@@ -62,12 +70,24 @@ def publish_recording_status(self, message: str) -> None:
 		status_topic = f"{str(DAShboard.recording_status_root)}/{self.device}"
 		self.client.publish(status_topic, message, retain=True)
 
+	def publish_camera_is_online(self, message: str) -> None:
+		""" Send a message on the current device's online topic
+		"""
+		status_topic = f"{str(Camera.online_root)}/{self.device}"
```

Comment:
Maybe set this as `online_status_topic` to make it more clearer?

Reply:
Done
1532e08